lib/rails_admin/config/actions/base.rb

```diff
@@ -78,6 +78,11 @@ class Base
           true
```

Comment:
```suggestion
          target_window.blank?
```
Or maybe `target_window != :_blank`